tests/DependencyInjection/Compiler/DoctrineMiddlewareCompilerPassTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DH\AuditorBundle\Tests\DependencyInjection\Compiler;
+
+use DH\Auditor\Configuration;
+use DH\Auditor\Provider\Doctrine\Auditing\Logger\Middleware\DHMiddleware;
+use DH\Auditor\Tests\Provider\Doctrine\Fixtures\Entity\Standard\Blog\Author;
+use DH\Auditor\Tests\Provider\Doctrine\Fixtures\Entity\Standard\Blog\Comment;
+use DH\Auditor\Tests\Provider\Doctrine\Fixtures\Entity\Standard\Blog\Post;
+use DH\Auditor\Tests\Provider\Doctrine\Fixtures\Entity\Standard\Blog\Tag;
+use DH\AuditorBundle\DependencyInjection\Compiler\DoctrineProviderConfigurationCompilerPass;
+use DH\AuditorBundle\DependencyInjection\DHAuditorExtension;
+use Doctrine\Bundle\DoctrineBundle\DependencyInjection\DoctrineExtension;
+use Doctrine\DBAL\Driver\Middleware;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * @internal
+ *
+ * @small
+ */
+final class DoctrineMiddlewareCompilerPassTest extends AbstractCompilerPassTestCase
+{
+    public function testCompilerPass(): void
+    {
+        if (!interface_exists(Middleware::class) || !class_exists(DHMiddleware::class)) {
+            self::markTestSkipped('DHMiddleware doesn\'t supported');
```

Comment:
Small typo here: **isn't** instead of **doesn't**

Reply:
done